code/modules/clothing/head/misc.dm

```diff
@@ -28,6 +28,28 @@
 	item_state = "that"
 	flags = FPRINT
 	siemens_coefficient = 0.9
+	var/magichatchance = 5 //%
+	var/ismagichat
+	var/timer
+
+/obj/item/clothing/head/that/New()
+	if(prob(magichatchance))
+		desc = "It's a magic looking hat."
+		ismagichat = TRUE
+		wizard_garb = TRUE
+
+/obj/item/clothing/head/that/attackby(obj/item/W, mob/user)
+	if(ismagichat)
+		var/allowed_wands = list(/obj/item/item_handle, /obj/item/weapon/cane, /obj/item/weapon/nullrod, /obj/item/weapon/staff)
+		if(is_type_in_list(W, allowed_wands))
+			if(world.time - timer >= 20SECONDS)
```

Comment:
I'm surprised `20SECONDS` without a space compiles

Reply:
Works on my machine. I can add one if it bothers your autism.